bootstrap/kubeadm/controllers/kubeadmconfig_controller.go

```diff
@@ -88,6 +88,8 @@ type Scope struct {
 
 // SetupWithManager sets up the reconciler with the Manager.
 func (r *KubeadmConfigReconciler) SetupWithManager(ctx context.Context, mgr ctrl.Manager, option controller.Options) error {
+	option.Log = mgr.GetLogger().WithValues("reconciler", "KubeadmConfigReconciler")
```

Comment:
```suggestion
	option.Log = mgr.GetLogger().WithValues("reconciler", "KubeadmConfig")
```

Seems might be redundant?